app/modules/import-export/usage.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Elementor\App\Modules\ImportExport;
+
+use Elementor\App\Modules\ImportExport\Processes\Revert;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Usage {
+
+	/**
+	 * Register hooks.
+	 *
+	 * @return void
+	 */
+	public function set_usage_data() {
+		add_filter( 'elementor/tracker/send_tracking_data_params', function ( array $params ) {
+			$params['usages']['import_export']['revert'] = $this->get_revert_usage_data();
+
+			return $params;
+		} );
+	}
+
+	/**
+	 * Get the Revert usage data.
+	 *
+	 * @return array
+	 */
+	private function get_revert_usage_data() {
+		$revert_sessions = ( new Revert() )->get_revert_sessions();
```

Comment:
I think it should be a dependency of this class and shouldn't be instantiated here

Reply:
Let's talk about it.